opencog/persist/zmq/events/AtomSpacePublisherModule.cc

```diff
@@ -365,6 +367,24 @@ Object AtomSpacePublisherModule::tvToJSON(TruthValuePtr tvp)
             break;
         }
 
+        case PROBABILISTIC_TRUTH_VALUE: {
+            json.push_back(Pair("type", "probabilistic"));
+            jsonDetails.push_back(Pair("strength", tvp->getMean()));
+            jsonDetails.push_back(Pair("count", tvp->getCount()));
+            jsonDetails.push_back(Pair("confidence", tvp->getConfidence()));
+            json.push_back(Pair("details", jsonDetails));
+            break;
+        }
+
+        case FUZZY_TRUTH_VALUE: {
+            json.push_back(Pair("type", "fuzzy"));
+            jsonDetails.push_back(Pair("strength", tvp->getMean()));
+            jsonDetails.push_back(Pair("count", tvp->getCount()));
+            jsonDetails.push_back(Pair("confidence", tvp->getConfidence()));
```

Comment:
It is suppose to store only strength and conf.

But there's also a count KKK that starts at 800, but the count is actually calculated from the conf.

Anyway, all TV type can output all 3 values, even STV.